### ISSUE TITLE
Routing support

### ISSUE THREAD

**Issue by heruan** [vaadin/vaadin-tabs-flow#22]
When I'm using `Tabs` in Flow, I'd like to link each `Tab` to a certain route, so that I can have each tab bound to a URL and the user can reach the wanted tab easily (e.g. bookmarks).

**Comment by pleku** [comment]
Blocked by https://github.com/vaadin/vaadin-tabs/issues/81

**Comment by cdir** [comment]
This is a very important issue as long as we need to deal without @PreserveOnRefresh!

**Comment by Legioth** [comment]
> Blocked by vaadin/vaadin-tabs#81

Not necessarily. The tabs don't have to be router links. The navigation logic can instead be hooked up through with a server-side tab listener that calls `Router.navigate`.

In either case, it would also be necessary to have reverse logic on the server that listens for navigation events and updates which tab is selected as appropriate.
